docs/developing.md

```diff
@@ -77,19 +77,67 @@ The Actor tests use resmoke to set up a real MongoDB cluster and execute
 the test binary. The resmoke yaml config files that define the different
 cluster configurations are defined in `src/resmokeconfig`.
 
-```sh
-# Example:
-./run-genny resmoke-test --suites src/resmokeconfig/genny_standalone.yml
-```
-
 Each yaml configuration file will only run tests that are associated
-with their specific tags. (Eg. `genny_standalone.yml` will only run
-tests that have been tagged with the `[standalone]` tag.)
+with their specific tags. (Eg. `genny_sharded.yml` will only run
+tests that have been tagged with the `[sharded]` tag.)
 
 When creating a new Actor, `./run-genny create-new-actor` will generate a new test case
 template to ensure the new Actor can run against different MongoDB topologies,
 please update the template as needed so it uses the newly-created Actor.
 
+Currently, the code to implement resmoke-test assumes that it will be run by
+Evergreen. Therefore, to run resmoke-test locally, some modifications must be made.
+The commands to make these modifications on MacOS and on Linux are detailed below.
+
+These commands assume that `git status` last output line is
+`nothing to commit, working tree clean`, and that the current working directory is
+`genny/` i.e. the root of the repo.
+
+Note: If you make any changes after having run `./run-genny install -d {your distro}`,
+please run that command again to compile your changes.
+
+#### MacOS:
+1. `mkdir -p data/db && sed -i '' '/mongos,/s!mongos,!mongos, "--dbpathPrefix", "data/db",!' src/lamplib/src/genny/tasks/run_tests.py`
+2. `sed -i '' '/program_executable/s!:.*$!: ../../build/src/cast_core/cast_core_test!' src/resmokeconfig/{YML file you want to use}`
+   ```sh
+   # Example:
+   sed -i '' '/program_executable/s!:.*$!: ../../build/src/cast_core/cast_core_test!' src/resmokeconfig/genny_sharded.yml
+   ```
+3. If using genny_create_new_actor.yml: `./run-genny create-new-actor SelfTestActor`. Fix the bug in the generated code, which is left as an exercise to ensure the reader understand the code.
```

Comment:
This seems to suggest that for other tests there is no error in the generated code. But actually all have the same error.

On the other hand, according to my test  genny_create_new_actor.yml does require the actor named `SelfTestActor` while `genny_single_node_replset.yml` does not care it. 

Therefore, for simplicity, I'd suggest remove "If using genny_create_new_actor.yml:", just tell people to create the "SelfTestActor"

Reply:
My understanding is that only `genny_create_new_actor.yml` requires a new actor to be generated, the other YML files don't require any. I reckon it's important to document that fact in some way.